src/Form/FormLabel.js

```diff
@@ -0,0 +1,63 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import LabelBase from '../internal/LabelBase';
+
+export const styleSheet = createStyleSheet('FormLabel', () => {
+  return {
+    root: {}, // expected to be used
```

Comment:
What's the plan with this style? Right now, we could simply be using `LabelBase` over `FormLabel`
.